apps/idam/idam-web-public/preview.yaml

```diff
@@ -4,6 +4,10 @@ metadata:
   name: idam-web-public
   namespace: idam
 spec:
+  securityContext:
```

Comment:
this won't do anything, doubt it's available inside the chart and everything in the chart is under values.java anyway